meta-dts-distro/recipes-kernel/kernel/linux-yocto_%.bbappend

```diff
@@ -4,7 +4,6 @@ SRC_URI:append = " \
     file://acpi-bgrt.cfg \
     file://dts.cfg \
     file://enable-memconsole-coreboot.cfg \
-    file://efivars.cfg \
```

Comment:
Do we use this file or not? If not then delete file too if yes then this line probably should be here.

Reply:
35d1227ef133879e42e9072991dc5d9616860f11